content-src/asrouter/templates/FullPageInterrupt/_FullPageInterrupt.scss

```diff
@@ -0,0 +1,213 @@
+.activity-stream {
+  &.welcome {
+    overflow: hidden;
+  }
+
+  &:not(.welcome) {
+    .fullpage-wrapper {
+      display: none;
+    }
+  }
+}
+
+.fullpage-wrapper {
+  $responsive-breakpoint: 850px;
+  $responsive-width: 352px;
+
+  align-content: center;
+  display: flex;
+  flex-direction: column;
+  overflow-x: auto;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 21000;
+
+  + div {
+    opacity: 0;
+  }
+
+  .fullpage-icon {
+    background-position-x: left;
+    background-repeat: no-repeat;
+    background-size: contain;
+
+    &:dir(rtl) {
+      background-position-x: right;
+    }
+
+    @media screen and (max-width: $responsive-breakpoint) {
+      background-position: center;
+    }
+  }
+
+  .brand-logo {
+    background-image: url('chrome://branding/content/about-logo.png');
+    margin: 20px 10px 10px 20px;
+    padding-bottom: 50px;
+  }
+
+  .welcome-title,
+  .welcome-subtitle {
+    align-self: center;
+    margin: 0;
+    @media screen and (max-width: $responsive-breakpoint) {
+      text-align: center;
+    }
+  }
+
+  .welcome-title {
+    color: $trailhead-purple-80;
+    font-size: 46px;
+    line-height: 62px;
+  }
+
+  .welcome-subtitle {
+    color: $trailhead-violet;
+    font-size: 20px;
+    line-height: 27px;
+  }
+
+  .container {
+    display: flex;
+    align-self: center;
+    padding: 40px 0;
+
+    @media screen and (max-width: $responsive-breakpoint) {
+      flex-direction: column;
+      width: $responsive-width;
+      text-align: center;
+    }
+  }
+
+  .fullpage-left-section {
+    position: relative;
+    width: 460px;
+    margin-inline-start: 10px;
+    margin-inline-end: 2px;
+    font-size: 18px;
+    line-height: 30px;
+
+    @media screen and (max-width: $responsive-breakpoint) {
+      width: $responsive-width;
+    }
+
+    .fullpage-left-content {
+      color: $grey-60;
```

Comment:
This doesn't look right to me, it's too dark compared to the comps

Reply:
Another thing I'm noticing is that it's a bit weird to have this be `normal` weight when the card text is `200`